warehouse/manage/views.py

```diff
@@ -39,6 +39,33 @@
 from warehouse.utils.project import confirm_project, destroy_docs, remove_project
 
 
+def user_projects(request):
+    """ Return all the projects for which the user is a sole owner """
+    projects_owned = (
+        request.db.query(Project)
+        .join(Role.project)
+        .filter(Role.role_name == "Owner", Role.user == request.user)
+        .subquery()
+    )
+
+    with_sole_owner = (
+        request.db.query(Role.package_name)
+        .join(projects_owned)
+        .filter(Role.role_name == "Owner")
+        .group_by(Role.package_name)
+        .having(func.count(Role.package_name) == 1)
+        .subquery()
+    )
+
+    return {
+        "projects_owned": request.db.query(projects_owned).all(),
+        "projects_sole_owned": request.db.query(Project)
```

Comment:
Small nit, can you reformat this line to be:

```python
"projects_sole_owned": (
    request.db.query(Project).join(with_sole_owner).order_by(Project.name).all()
),
```

Reply:
Updated